### DISCUSSION TITLE
Minibuffer/floating window location configurable?

### DISCUSSION THREAD

**Discussion by karbak** [lem-project/lem#661] [Q&A] [answered]
Hello - thanks for a great 2.0 release!  Starting to use lem more regularly, I miss the bottom pane location of the minibuffer that I'm accustomed to after decades of Emacs use.  Is there any way to configure this? Edit: I did root around the source, but nothing obvious stands out.

**Comment by cxxxr** [comment, marked as answer]
Hi, I am very happy that you are using lem.
Also, your feedback is very helpful.

Unfortunately, it does not currently offer such functionality.
First of all, I am a little reluctant to make lem exactly the same as GNU emacs.
This is because it would be taking legacy parts of emacs. (Of course that can incorporate the good parts too).

Another problem is that the eye movement is greater.
For example, if you are using a 40" display, the lower left corner is very difficult to see.

However, I feel that the current UI also has many areas for improvement.
For example, "Abort", "End of buffer", and other unimportant messages are also placed on top of the code, preventing it from being displayed.
In this respect, minibuffer seems to have the advantage.

**Comment by aykaramba** [comment]
I too expect the buffer popup on the bottom but I am starting to get used to the popup, I appreciate the thought put into questioning the status quo.  I use a 65" tv as a monitor and you are probably right that reducing eye travel might be a good idea.

Looking forward to testing it some more.
